pom.xml

```diff
@@ -271,6 +271,24 @@
     <ivy.version>2.5.3</ivy.version>
     <c3p0.version>0.11.0-pre2</c3p0.version>
     <mchange-commons-java.version>0.3.2</mchange-commons-java.version>
+    <checker-qual.version>3.49.2</checker-qual.version>
+    <groovy-all.version>2.4.21</groovy-all.version>
+    <xml-apis.version>2.0.2</xml-apis.version>
+    <RoaringBitmap.version>1.3.0</RoaringBitmap.version>
+    <jmx_prometheus_javaagent.version>0.19.0</jmx_prometheus_javaagent.version>
```

Comment:
Suggest replacing this with `prometheus-jmx-exporter-javaagent` for naming consistency and to make prometheus front-and-center.